src/main/java/net/glowstone/generator/structures/GlowTemple.java

```diff
@@ -26,8 +26,6 @@ public GlowTemple(World world, int chunkX, int chunkZ) {
     public GlowTemple(World world, Random random, int chunkX, int chunkZ) {
         this(world, chunkX, chunkZ);
 
-        types.put(Biome.OCEAN, TempleType.values()[random.nextInt(3)]); // fix for lack of biomes
```

Comment:
Sorry, you're the expert here, but would you not need to do a new types.put statement to specify the biomes temples can spawn in?